TLM/TLM/Manager/Impl/SpeedLimitManager.cs

```diff
@@ -776,20 +749,26 @@ public class SpeedLimitManager : AbstractGeometryObservingManager, ICustomDataMa
 			foreach (Configuration.LaneSpeedLimit laneSpeedLimit in data) {
 				try {
 					if (!Services.NetService.IsLaneValid(laneSpeedLimit.laneId)) {
-						Log._Debug($"SpeedLimitManager.LoadData: Skipping lane {laneSpeedLimit.laneId}: Lane is invalid");
+						Log._Debug($"SpeedLimitManager.LoadData: Skipping lane {laneSpeedLimit.laneId}: " +
+						           $"Lane is invalid");
 						continue;
 					}
 
 					ushort segmentId = Singleton<NetManager>.instance.m_lanes.m_buffer[laneSpeedLimit.laneId].m_segment;
 					NetInfo info = Singleton<NetManager>.instance.m_segments.m_buffer[segmentId].Info;
-					int customSpeedLimitIndex = GetCustomNetInfoSpeedLimitIndex(info);
-					Log._Debug($"SpeedLimitManager.LoadData: Handling lane {laneSpeedLimit.laneId}: Custom speed limit index of segment {segmentId} info ({info}, name={info?.name}, lanes={info?.m_lanes} is {customSpeedLimitIndex}");
-					if (customSpeedLimitIndex < 0 || AvailableSpeedLimits[customSpeedLimitIndex] != laneSpeedLimit.speedLimit) {
+					var customSpeedLimit = GetCustomNetInfoSpeedLimit(info);
+					Log._Debug($"SpeedLimitManager.LoadData: Handling lane {laneSpeedLimit.laneId}: " +
```

Comment:
How many debug statements are shortened? I'm mixed feelings about this change. What does @krzychu124 / @VictorPhilipp think about this?

On one hand, I prefer the long one-line statements as it doesn't break what's going to be outputted. On the other hand, breaking it keeps them shorter in general.